subsys/bluetooth/shell/mcc.c

```diff
@@ -751,6 +799,21 @@ int cmd_mcc_read_current_track_obj_id(const struct shell *sh, size_t argc,
 	return result;
 }
 
+int cmd_mcc_set_current_track_obj_id(const struct shell *sh, size_t argc,
+				      char *argv[])
+{
+	int result;
+	uint64_t id = strtoul(argv[1], NULL, 0);
+
+	id = id & 0x0000FFFFFFFFFFFF; /* 48 bits only */
```

Comment:
```suggestion
	id = id & 0x0000FFFFFFFFFFFFUL; /* 48 bits only */
```

Reply:
Since this is a value that is used multiple places, suggest to add a #define for it.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/38528

Reply:
Fixed three places.
#define suggested in upsteream PR.  (Should be in OTS.)